proteus/gmsh2triangle.py

```diff
@@ -0,0 +1,107 @@
+import numpy as np
+from proteus.MeshTools import Edge
+from proteus.Profiling import logEvent
+
+def msh2triangle(fileprefix):
```

Comment:
I think this could go into MeshTools.  Needs a docstring here, but I assume you're just converging a gmsh .msh file to a triangle format mesh, right?